fingerprint2.js

```diff
@@ -111,7 +111,7 @@
         var values = [];
         that.each(newKeys, function(pair) {
           var value = pair.value;
-          if (typeof pair.value.join !== "undefined") {
+          if (pair.value && typeof pair.value.join !== "undefined") {
```

Comment:
http://www.youku.com/
Visit this site in mobile emulation mode in Chrome, and screen object will be a String rather than an Object, which is set in html inline script.
So every `pair.value` get from `screen` will be undefined.